hotels.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href='index.html'><img src="img.img" width="138" height="70" alt="Логотип Sedona"></a>
```

Comment:
```suggestion
        <a href="index.html"><img src="img.img" width="138" height="70" alt="Логотип Sedona"></a>
```

Критерий [Б15](https://up.htmlacademy.ru/htmlcss/28/criteries#b15) (другой тип кавычек).